### PR TITLE
Focus comment or group event after adding it

### PR DESCRIPTION
**Pull request by AlexandreSi**
Different things:

- Fix bug that added a new line when one typed Enter key to edit a comment event
- Focus comment or group event after adding it (so that one doesn't have to click on it after adding it)
- Keep the same style when editing a group or comment event
- Submit modifications of group or comment event with Ctrl/Cmd + Enter

Before:
image

After:
image

### CODE REVIEW

File: newIDE/app/src/UI/TextField.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ type Props = {|
   margin?: 'none' | 'dense',
   fullWidth?: boolean,
   style?: {|
-    fontSize?: 14 | 18 | '1.3em',
+    fontSize?: 14 | 18 | '1.3em' | 'inherit', // 'inherit' should only be used on an event sheet where font size is adapted to zoom
```
Comment (4ian) [inline newIDE/app/src/UI/TextField.js:97]: Very useful explanation.

File: newIDE/app/src/EventsSheet/EventsTree/style.css [hunk 2]
```diff
@@ -80,6 +80,10 @@
     font-weight: normal;
 }
 
+.gd-events-sheet .rst__rowTitle .MuiInputBase-root {
+    font-family: unset;
```
Comment (4ian) [inline newIDE/app/src/EventsSheet/EventsTree/style.css:84]: Maybe add a comment telling that this is for comments and groups.

**Review by 4ian — APPROVED** [review]

**Comment by Bouh** [comment]
Is there any small improvement that can suggest?

- In the past I had made the the font display and especially the spacing and size of comments and groups do not change between the basic display and editing.
The display on groups is good, but on comments the height is higher when editing.

- image

The blue line in edit mode on comment is almost invisible.

Thx for these improvements, the focus and enter key is neat!

**Comment by AlexandreSi** [comment]
> The display on groups is good, but on comments the height is higher when editing.

Fixed

> The blue line in edit mode on comment is almost invisible.

I wonder if the simplest solution would be to just remove this line. Wdyt?

**Comment by Bouh** [comment]
> > The blue line in edit mode on comment is almost invisible.
> 
> I wonder if the simplest solution would be to just remove this line. Wdyt?

Yes 👍

**Comment by 4ian** [comment]
This is becoming really good ✨

**Comment by AlexandreSi** [comment]
Ok, now there should absolutely no difference between the 2 states (editing and not editing)

**Comment by Bouh** [comment]
> Ok, now there should absolutely no difference between the 2 states (editing and not editing) Test here : https://gdevelop-storybook.s3.amazonaws.com/improve-some-events-component/latest/index.html?path=/story/eventssheet-eventssheet--default-no-scope

You are the best 💯

**Pull request merged**